jetson/ra_kinematics/README.md

```diff
@@ -0,0 +1,113 @@
+
+### Usage ###
+
+To build the ra_kinematics package, run `$ ./jarvis build jetson/ra_kinematics/ ` from the mrover-workspace directory.
+
+To run the ra_kinematics package, run `$ ./jarvis exec jetson/ra_kinematics/ ` from the mrover-workspace directory.
+
+Currently, output is logged to stdout.
+
+#### Additional Usage ####
+
+If you are using the ra_kinematics package, you may also wish to run the associated GUI, which is found in the base_station/kineval_stencil/ directory. Alternatively, you may use base_station/gui/ to access lcm_echo. To build and run either, run \
+`$ ./jarvis build base_station/kineval_stencil/ ` and `$ ./jarvis exec base_station/kineval_stencil/ ` \
+or \
+`$ ./jarvis build base_station/kineval_stencil/ ` and `$ ./jarvis exec base_station/kineval_stencil/ ` \
+from the mrover-workspace directory.
+
+Also, be sure to build and run lcm_bridge by running \
+`$ ./jarvis build lcm_bridge/server/ ` and `$ ./jarvis exec lcm_bridge/server/ ` \
+from the mrover-workspace directory.
+
+### Known Possible Errors ###
+
+If MotionPlanner is unable to find a path, undefined behavior could arise when trying to access points along the path, though this has not been witnessed. This should be addressed soon.
+
+No other error conditions are currently known.
+
+### Testing ###
+
+The ra_kinematics package uses the EECS 280 testing framework. Testing files are found in the test directory.
+
+To test the package, run `$ make tests ` from the ra_kinematics directory. To test more specific files, see the Makefile for more commands.
+
+### LCM Publications ###
+
+#### Arm Position \[Publisher\] "/arm_position" ####
+Message: [ArmPosition.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/ArmPosition.lcm) \
+Publisher: jetson/ra_kinematics \
+Subscriber: jetson/ra_kinematics
+
+When in Simulation Mode, jetson/ra_kinematics publishes to /arm_position to simulate receiving arm position message from nucleo_bridge.
+
+#### Debug Message \[Publisher\] "/debug_message" ####
+Message: [DebugMessage.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/DebugMessage.lcm) \
+Publisher: jetson/ra_kinematics \
+Subscriber: base_station/kineval_stencil
+
+#### RA Closed Loop \[Publisher\] "/ik_ra_control" ####
+Message: [ArmPosition.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/ArmPosition.lcm) \
+Publisher: jetson/ra_kinematics \
+Subscriber: jetson/nucleo_bridge
+
+#### FK Transform \[Publisher\] "/fk_transform" ####
+Message: [FKTransform.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/FKTransform.lcm) \
+Publisher: jetson/ra_kinematics \
+Subscriber: base_station/kineval_stencil
+
+### LCM Subscriptions ###
+
+#### Arm Position \[Subscriber\] "/arm_position" ####
+Message: [ArmPosition.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/ArmPosition.lcm) \
```

Comment:
what is the different between this & /ik_ra_control ?

Reply:
/ik_ra_control is the target set of angles that we send to ESW. /arm_position is the set of angles that ESW sends us, which is where the arm is currently at.

We send to /arm_position instead of /ik_ra_control when in sim mode, since we're essentially simulating both us sending the target to ESW and ESW sending back encoder readings that exactly hit the target. The Readme can be updated to make this more clear